one_collect/src/helpers/exporting/modulemetadata.rs

```diff
@@ -117,6 +118,9 @@ impl ElfModuleMetadata {
         &mut self,
         build_id: Option<&[u8; 20]>) {
         self.build_id = build_id.copied();
+        if build_id.is_some() {
+            debug!("ELF build-id set");
```

Comment:
Rather than logging inside of `ElfModuleMetadata`, instead you should log inside of `load_elf_metadata`.  Then, you can log this information in the context of the file being processed.